chapters/sampling-randomization-power.tex

```diff
@@ -1,529 +1,788 @@
+	\textbf{Treatment:} The general word for the evaluated intervention or event.
+	This includes being offered training or cash transfer from a program, experiencing a natural disaster etc.}
+against a counterfactual control group.\sidenote{
+	\textbf{Counterfactual:} A statistical description of what would have happened
+	to specific individuals in an alternative scenario,
+	for example, a different treatment assignment outcome.}
+\index{counterfactual}
+The key assumption is that each
+person, facility, or village (or whatever the unit of intervention is)
+had two possible states: their outcome if they did receive the treatment
+and their outcome if they did not receive that treatment.
+The average impact of the treatment is defined as
+the difference between these two states averaged over all units.
+
+However, we can never observe the same unit
+in both the treated and untreated state simultaneously,
+so we cannot calculate these differences directly.
+Instead, the treated group is compared to a control group
+that is statistically indistinguishable,
+which makes the average impact of the treatment
+mathematically equivalent to the difference in averages between the groups.
+Statistical similarity is often defined
+as \textbf{balance} between two or more groups.	
+This test is so common,	
+DIME Analytics created the Stata command \texttt{iebaltab}\sidenote{	
+	\url{https://dimewiki.worldbank.org/iebaltab}}	
+to generates tables of balance checks	
+as part of the package \texttt{ietoolkit}.	
+However, each research design has its different method	
+for how the statistically similar control group is identified.
+
+The rest of this section covers how the data requirement differs 
+between different research designs.
+What does not differ, however, is that the authoritative source
+for which unit is a treated unit and which is a control unit
+should always be one or several variables in your master dataset. 
+You will often have to merge that data to other datasets, 
+but that is an easy task if you created a data map.
+
+%%%%% Experimental design
+
+In \textbf{experimental research designs},\sidenote{
+	For example, \textbf{randomized control trials (RCTs) --}
+	\url{https://dimewiki.worldbank.org/Randomized\_Control\_Trials}}
+\index{randomized control trials} \index{experimental research designs}
+the research team determines which members
+of the studied population will receive the treatment.
+This is typically done by a randomized assignment process
+in which a subset of the eligible population
+is randomly selected to receive the treatment
+(see later in this chapter for how to implement this).
+The intuition is that if everyone in the eligible population
+is assigned at random to either the treatment or control group,
+then the two groups will, on average, be statistically indistinguishable,
+The treatment will therefore not be correlated with anything
+but the impact of that treatment.\cite{duflo2007using}
```

Comment:
I read this a more general way of describing "having received treatment". Since not all treatments are "received". Living across the street from a different school district is a type of treatment assignment, but no one "received" anything there. Or farmers that experiences a natural disaster. I think "received" is very much associated with RCT types of treatment. 

But if you are confused by this sentence, than many people will.